### ISSUE TITLE
Issue Targeting x86 Platform

### ISSUE THREAD

**Issue by hatboyzero** [NLua/NLua#67]
nuget.package/tools/net45/GetLibLuaPostBuildCmd.ps1

This file adds post build events that copy the wrong version of the native library for our project.  We are developing on an x64 machine, but targeting x86 processors.  Because the machine is x64, the install script assumes, incorrectly, that we desire the x64 library, and we get a Bad Image Format exception when we try to run the code.

Also, the path of the reference in the post-build event is hard coded to the development machine, which will result in a failure on our build server.

Maybe this is the same culprit as what is described in #66.

**Comment by nrother** [comment]
Is there any progress on updating the NuGet Package? I'd really like to add a dependency to NLua from my [DynamicLua](https://github.com/nrother/dynamiclua) instead of including it, but this would require a version that has this bug fix included ;)

**Comment by viniciusjarina** [comment]
@nrother This was fixed a long time ago.

Is not working for you?

**Comment by gregwmay** [comment]
Eight years later... I downloaded the most recent NuGet packages (1.6.3) which includes the KeraLua dependency (1.3.4). I built a test application and it ran perfectly the first time, but on the second and subsequent attempts I got the System.BadImageFormatException exception described earlier in this issue. I am running a Windows Forms application on .NET 4..8 framework and targeting AnyCPU.

My fix was to uncheck the "Prefer 32-bit" option in the Compile properties of my Windows Forms project. With that off, it works reliably.

I am currently only building on a dev machine so this may come back to bite me at release time, but as of today that was my fix.
